sips/sip-026/sip-026-clarity-defi-vault.md

```diff
@@ -0,0 +1,97 @@
+# Preamble
+
+SIP-Number: SIP-026
+Title: Clarity DeFi Vault
+Author: Tycho Onnasch, Fernando Foy, Philip de Smedt, Christian Hresko
+Consideration: Technical
+Type: Standard
+Status: Accepted
+Created: Sep 7, 2023
+Last-Modified: Sep 7, 2023
+Sign-off:
+Discussions-To: https://forum.stacks.org/t/clarity-defi-vault-sip/15567
+License: Creative Commons CC0 1.0 Universal license
+Layer: Trait
+
+# Abstract
+
+This Stacks Improvement Proposal (SIP) aims to address the issue of commingled collateral positions on the Stacks blockchain's Clarity-based DeFi applications. Currently, due to Clarity's design, it is challenging for users to identify their specific collateral positions within the Stacks Explorer. This SIP proposes the creation of a common interface for contracts that hold SIP-010 assets. Implementing this interface will enable users to view their collateral positions distinctly, improving the overall user experience of Stacks DeFi applications.
+
+# License and Copyright
+
+This SIP is made available under the terms of the Creative Commons CC0 1.0 Universal license, available at https://creativecommons.org/publicdomain/zero/1.0/. This SIP's copyright is held by the Stacks Open Internet Foundation.
+
+# Introduction
+
+#### Problem Statement
```

Comment:
When mentioning how Ethereum has resolved this problem, can you add a link to an exmple contract/etherscan tx etc?

Reply:
Here's an example of a pool deployed by a transaction: https://etherscan.io/address/0xc1dd3f011290f212227170f0d02f511ebf57e433#code

Users can mint, burn and deploy assets to the pool and can follow the assets stored in the pool by looking at the balance in the explorer.